samples/algorithms/repeat-until-success/RepeatUntilSuccess.qs

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Samples.RepeatUntilSuccess {
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Preparation;
+    open Microsoft.Quantum.Diagnostics;
+
+    /// # Summary
+    /// Example of a Repeat-until-success algorithm implementing a circuit 
+    /// that achieves exp(i⋅ArcTan(2)⋅Z) by Paetznick & Svore. 
+    /// Gate exp(i⋅ArcTan(2)⋅Z) is also know as V gate.
+    /// # References
+    /// - [ *Adam Paetznick, Krysta M. Svore*,
+    ///     Quantum Information & Computation 14(15 & 16): 1277-1301 (2014)
+    ///   ](https://arxiv.org/abs/1311.1074)
+    /// For circuit diagram, see file RUS.png (to be added to README).
+    ///
+    /// The program executes a circuit on a "target" qubit using an "auxiliary" and 
+    /// "resource" qubit. The circuit consists of two parts (red and blue in image).
+    /// The goal is to measure Zero for both the auxiliary and resource qubit.
+    /// If this succeeds, the program will have effectively applied an 
+    /// Rz(arctan(2)) gate (also known as V_3 gate) on the target qubit.
+    /// If this fails, the program reruns the circuit up to <limit> times.
+    @EntryPoint()
+    operation ApplyRzArcTan2(
+        inputValue : Bool,
+        inputBasis: Pauli,
+        limit: Int
+    ) : (Bool, Result, Int) {
+        using ((auxiliary, resource, target) = (Qubit(), Qubit(), Qubit())) {
+            // Prepare auxiliary and resource qubits in |+〉state
+            SetXZeroFromOne(auxiliary);
+            SetXZeroFromOne(resource);
+            /// Prepare target qubit in |0> or |1> state, depending on input value
```

Comment:
```suggestion
            // Prepare target qubit in |0〉or |1〉 state, depending on input value
```
There are some more occurrences of `///` comments and places where unicode can be useful in the file, just giving one as an example.

Reply:
I believe @cgranade actually asked me to put these in, perhaps we should discuss what we want to do before I change everything back.

Reply:
Apologies for the lack of clarity on my part. In any case, `//` comments should be used for prose as @msoeken says, while `///` comments should be used for API documentation. The latter get picked up by documentation generation tools and by intelliSense when they immediately precede a namespace, operation, function, or user-defined type declaration, but have no special meaning elsewhere. For that reason, I suggest using `//` comments  rather than `///` comments in contexts where documentation comments aren't expected by the compiler (e.g.: within a callable body, between callable declarations, and so forth).

Both are useful in a sample, and serve different purposes, in that `//` comments help establish the prose for a sample, while `///` comments provide contextual reference information.

Reply:
Thanks, that makes sense. Posting these docs here for reference: https://docs.microsoft.com/en-us/quantum/user-guide/using-qsharp/file-structure#documentation-comments